ibm_dsx directory , updated readme

sathayen opened the pull request:
* The `ibm_dsx` directory has a script to automatically convert the all example notebooks to make them compatible to run directly on IBM DSx. The original example notebooks remain unchanged.. the script only makes a copy of the original notebook and updates this copy. 

* Also added a new section in `README.md` for IBM DSx and updated the contributors list.

ewinston commented:
Hi Ninad,
There are some links in the text which point to your fork but should probably point to the main repo. This means we will need to merge your wiki page first. On your wiki could you clone it locally and add the images to the wiki repo instead of the main repo? Finally we are also trying to standardize on RST instead of Markdown. Can you convert your markdown with e.g. pandoc? 
Thanks,
Erick

sathayen commented:
Hi Erick, 

I have made the necessary changes. Please review. 

1. Updated the URL to the DSx instructions in the main README.rst file . Please note that the page I have mentioned is the following: https://github.com/qiskit-tutorial/wiki/Running-Quantum-Program-on-IBM-DSx. It does not exist as of now and needs to be created (using number 3 below) 

2. Changed all the markdown files to rst  (.rest for wiki) 

3. Moved the wiki specific images into a new "images" directory under my wiki git repo -- git@github.com:sathayen/qiskit-tutorial.wiki.git . So I hope we can just clone this repo and merge it into the main qiskit-tutorial  wiki. However, I am not sure if I can make a pull request for the 'wiki' git repo. 

4. The wiki instructions are updated to .rest format and all image references are local to this (e.g. .. image::images/ds1.png)
